customization/parallel-data.md

```diff
@@ -12,7 +12,13 @@ description: Parallel data for machine translation
 
 Parallel datasets can include translations for one or more language pairs, and be directioned or directionless.
 
-Parallel data can be created by crawling and aligned monolingual test, and by [back-translation](/customization/back-translation.md) or [back-copying](/customization/back-translation.md).
+### Parallel data creation
+
+Parallel datasets can be created manually, automatically, or created synthetically from monolingual data.
+- Human translation or human [post-editing](/workflows/post-editing)
+- [Crawling](/research/crawling), [alignment](/research/alignment) and [filtering](/customization/filtering)
+- [Back-translation](/customization/back-translation.md)
+- [Back-copying](/customization/back-translation.md)
```

Comment:
@cefoo Maybe Filtering and Back-translation belong in something called datasets/, not customization/?

Reply:
We have a [Building](https://github.com/machinetranslate/machinetranslate.org/blob/master/ROADMAP.md#building) section. Do you think they can fit there?

Reply:
It's another option.